views/Users/List/List.module.scss

```diff
@@ -1,11 +1,16 @@
 @import 'colors';
+@import 'layout';
```

Comment:
Try to put them in one line

@import 'colors', 'layout';

Reply:
Done